File: pinyougou-parent/pinyougou-shop-web/src/main/webapp/js/service/seckillOrderService.js
//服务层
app.service('seckillOrderService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.findAll=function(){
		return $http.get('../seckillOrder/findAll.php');
	};
	//分页 
	this.findByPagination=function(page,rows){
		return $http.get('../seckillOrder/findByPagination.php?page='+page+'&rows='+rows);
	};
	//查询实体
	this.findOne=function(id){
		return $http.get('../seckillOrder/findOne.php?id='+id);
	};
	//增加 
	this.add=function(entity){
		return  $http.post('../seckillOrder/add.php',entity );
	};
	//修改 
	this.update=function(entity){
		return  $http.post('../seckillOrder/update.php',entity );
	};
	//删除
	this.dele=function(ids){
		return $http.get('../seckillOrder/delete.php?ids='+ids);
	};
	//搜索
	this.search=function(page,rows,searchEntity){
		return $http.post('../seckillOrder/findWithConditionsByPagination.php?page='+page+"&rows="+rows, searchEntity);
	}    	
});

File: pinyougou-parent/pinyougou-sellergoods-interface/src/main/java/cn/porkchop/pinyougou/sellergoods/service/SeckillOrderService.java
package cn.porkchop.pinyougou.sellergoods.service;

import cn.porkchop.pinyougou.pojo.PageResult;
import cn.porkchop.pinyougou.pojo.TbSeckillOrder;

import java.util.List;
/**
 * 服务层接口
 * @author Administrator
 *
 */
public interface SeckillOrderService {

	/**
	 * 返回全部列表
	 * @return
	 */
    List<TbSeckillOrder> findAll();
	
	
	/**
	 * 返回分页列表
	 * @return
	 */
    PageResult findByPagination(int pageNum, int pageSize);
	
	
	/**
	 * 增加
	*/
    void add(TbSeckillOrder seckillOrder);
	
	
	/**
	 * 修改
	 */
    void update(TbSeckillOrder seckillOrder);
	

	/**
	 * 根据ID获取实体
	 * @param id
	 * @return
	 */
    TbSeckillOrder findById(Long id);
	
	
	/**
	 * 批量删除
	 * @param ids
	 */
    void delete(Long[] ids);

	/**
	 * 分页
	 * @param pageNum 当前页 码
	 * @param pageSize 每页记录数
	 * @return
	 */
    PageResult findWithConditionsByPagination(TbSeckillOrder seckillOrder, int pageNum, int pageSize);
	
}
